app/presenters/admin_presenter.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# The page model for the admin page
+class AdminPresenter
+  attr_accessor :work_stats
```

Comment:
Can the work stats be set as part of the initializer?

Reply:
Done.